crates/fluvio-version-manager/src/command/switch/mod.rs

```diff
@@ -0,0 +1,94 @@
+//! Version Switching Command
+//!
+//! The `switch` command is responsible of changing the active Fluvio Version
+
+mod version_directory;
+
+use anyhow::Result;
+use clap::Parser;
+use colored::Colorize;
+
+use fluvio_hub_util::fvm::Channel;
+
+use crate::GlobalOptions;
+use crate::common::settings::Settings;
+use crate::common::workdir::fvm_versions_path;
+use crate::common::notify::Notify;
+
+use self::version_directory::VersionDirectory;
+
+#[derive(Debug, Parser)]
+pub struct SwitchOpt {
+    #[command(flatten)]
+    global_opts: GlobalOptions,
```

Comment:
Why global option is here? This should be moved to root command

Reply:
This is used by the `Notify` trait to access `is_quiet` global option, which supress output.

Reply:
I assume this is needed to be repeated for every command.  either remove or move to root.  for example this could be passed from root cmd